tutorials/raw/plot_events.py

```diff
@@ -0,0 +1,153 @@
+#
+#
+# Mapping Event IDs to trial descriptors
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# So far in this tutorial we've only been dealing with integer Event IDs, which
+# were assigned based on DC voltage pulse magnitude (which is ultimately
+# determined by the experimenter's choices about what signals to send to the
+# STIM channels). Keeping track of which Event ID corresponds to which
+# experimental condition can be cumbersome, and it is often desirable to pool
+# experimental conditions during analysis. You may recall that the mapping of
+# integer Event IDs to meaningful descriptions for the :ref:`sample dataset
+# <sample-dataset>` is given in :ref:`this table
+# <sample-data-event-dict-table>` in the :doc:`introductory tutorial
+# <../intro/plot_introduction>`. Here we simply reproduce that mapping as an
+# *event dictionary*:
+
+event_dict = {'auditory/left': 1, 'auditory/right': 2, 'visual/left': 3,
+              'visual/right': 4, 'smiley': 5, 'buttonpress': 32}
+
+###############################################################################
+# Event dictionaries like this one are used when extracting epochs from
+# continuous data, and the resulting :class:`~mne.Epochs` object allows pooling
+# by requesting partial trial descriptors. For example, if we wanted to pool
+# all auditory trials, instead of merging Event IDs 1 and 2 using the
+# :func:`~mne.merge_events` function, we can make use of the fact that the keys
+# of ``event_dict`` contain multiple trial descriptors separated by ``/``
+# characters: requesting ``'auditory'`` trials will select all epochs with
+# Event IDs 1 and 2; requesting ``'left'`` trials will select all epochs with
+# Event IDs 1 and 3. An example of this is shown in a later tutorial.
+#
+# .. TODO replace above sentence when the relevant tut is ready:
+#    An example of this is shown later, in the :ref:`epoch-pooling` section of
+#    the :ref:`epochs-intro-tutorial` tutorial.
+#
+#
+# Plotting events
+# ^^^^^^^^^^^^^^^
+#
+# Another use of event dictionaries is when plotting events, which can serve as
+# a useful check that your event signals were properly sent to the STIM
+# channel(s) and that MNE-Python has successfully found them. The function
+# :func:`mne.viz.plot_events` will plot each event versus its sample number
+# (or, if you provide the sampling frequency, it will plot them versus time in
+# seconds). It can also account for the offset between sample number and sample
+# index in Neuromag systems, with the ``first_samp`` parameter. If an event
+# dictionary is provided, it will be used to generate a legend:
+
+fig = mne.viz.plot_events(events, sfreq=raw.info['sfreq'],
+                          first_samp=raw.first_samp, event_id=event_dict)
+fig.subplots_adjust(right=0.7)  # make room for legend
+
+###############################################################################
+# Events can also be plotted alongside the :class:`~mne.io.Raw` object they
+# were extracted from:
```

Comment:
I think this has entity enough to appear in the index. I think I would do a subtitle to give the possibility to go directly to this point from the top of the document:

```
- Obtaining events from a raw object
- Reading and writing events to file
- Subselecting and combining events
- Mapping Event IDs to trial descriptors
- Plotting events
    - Overlay events in `raw.plot`
```